src/themes/teams/components/Icon/svg/icons/bold.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import * as cx from 'classnames'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg viewBox="0 0 32 32" role="presentation" className={classes.svg}>
+      <g className="icons-default-fill">
```

Comment:
Not sure if we need this class for anything.

Reply:
We don't need this for icons in Stardust. I am going to be importing all the icons without this class